camel-idea-plugin/src/test/resources/testData/CompleteJavaBeanRoute6TestData.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.main.Main;
+import testData.CompleteJavaBeanTestData;
+
+public final class CompleteJavaBeanRoute6TestData extends RouteBuilder {
+
+
+    @Override
+    public void configure() {
+        from("file:inbox")
+            .bean(beanTestData, "<caret>")
```

Comment:
How can this code compile? Should it not be a .class type or something, as `beanTestData` is not a field

Reply:
The purpose of the test is to test how the code behaves if there is an invalid field reference. So it's on purpose the beanTestData is not a field in this case